### python_conflict
QA Guru 03

Проект с нуля локально. 
Пройден флоу от git init до git push -u Добился возникновения конфликта, репозиторий с конфликтом, блокирующем мерж pullrequest-а)
